k8/debeziumservers.debezium.io-v1.yml

```diff
@@ -34,6 +34,7 @@ spec:
                         type: object
                         x-kubernetes-preserve-unknown-fields: true
                       type:
+                        default: json
```

Comment:
can you please rerun the `k8update` profile? Without using the `@Default` annotation I don't think v1 generator should actually produce this -- since it should ignore the default from `JsonProperty` based on the info from fabric8 guys.

Reply:
Good catch. Fixed.